curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md

```diff
@@ -942,3 +942,462 @@ Which of the following elements is used as a fallback for browsers lacking suppo
 #### --answer--
 
 `rp`
+
+## --quiz--
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `header` element in HTML?
+
+#### --distractors--
+
+It defines the main content of the document.
+
+---
+
+It represents a section with navigation links.
+
+---
+
+It defines a sidebar for the document.
+
+#### --answer--
+
+It marks the header of a document or section.
+
+
+### --question--
+
+#### --text--
+
+Which element is used to contain the main content of a web page?
+
+#### --distractors--
+
+`nav`
+
+---
+
+`article`
+
+---
+
+`section`
+
+#### --answer--
+
+`main`
+
+### --question--
+
+#### --text--
+
+What is the correct use of the `nav` element?
+
+#### --distractors--
+
+To define the header of a document.
+
+---
+
+To display metadata about a webpage.
+
+---
+
+To contain the main content of the page.
+
+#### --answer--
+
+To represent a section with navigation links.
+
```

Comment:
please also remove this extra line

please check all answers and remove the extra empty line after them

Reply:
here after line 1013 there are two empty lines, 1014 and 1015, there should not be two consecutive empty lines, please remove the extra empty line from here and from after all the answers that have an extra empty line after them